### DISCUSSION TITLE
Mount lib/routes in the docker image

### DISCUSSION THREAD

**Discussion by K33k0** [DIYgod/RSSHub#16649] [Q&A]
### What feature is it?

I'd like to develop routes inside of my existing docker instance. Mounting the routes folder would help with this

### What problem does this feature solve?

Currently I'm unable to find where the routes are stored inside the container.

### Additional description

If this is already available could you tell me the location, if not I'd love to be able to mount that folder

### This is not a duplicated feature request or new RSS proposal

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this feature has not already been requested and this is not a [new RSS proposal](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_en.yml).

**Comment by TonyRL** [comment]
The docker image `diygod/rsshub` is not suitable for development at all. Please setup a proper [node environment](https://docs.rsshub.app/joinus/new-rss/prerequisites) instead. If you really want to develop inside a container, you can check out the [cloud hosted development environment](https://docs.rsshub.app/joinus/new-rss/prerequisites#cloud-hosted-development-environment) section as well.

**Comment by wrobelda** [comment]
The original question is still valid for another use case: in case I wanted to use docker with my custom route that I built for my own purposes and didn't want to upstream it.
